boot/freeldr/freeldr/arch/uefi/arch/amd64/misc.S

```diff
@@ -0,0 +1,23 @@
+#include <asm.inc>
+#include <arch/pc/x86common.h>
+#include <arch/pc/pcbios.h>
+
+.code64
+
```

Comment:
Note: This helper file could be moved into the existing "boot/freeldr/freeldr/arch/amd64" directory instead, and you could move the definition for `__fastfail` from
https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/arch/amd64/entry.S;h=758eae4b677f3f3fb35c93a981c27cacb1eb26b0;hb=HEAD#l215
into it as well. (And in addition, remove those definitions from that entry.S file.)